api/transactions.js

```diff
@@ -224,14 +228,15 @@ function setTransactionBitFlags(transaction, options) {
  *
  * @param {RippleAddress} account
  * @param {Hex-encoded String|ASCII printable character String} identifier
+ * @param {Object} options
  * @param {Function} callback
  *
  * @callback
  * @param {Error} error
  * @param {Transaction} transaction
  */
-function getTransaction(api, account, identifier, callback) {
-  var options = {};
+function getTransaction(api, account, identifier, options, callback) {
+  assert.strictEqual(typeof options, 'object');
```

Comment:
Lower in the code, `options` is being mutated, which can lead to bugs if we are passing in an `options` with properties that are being set deep in the function. Maybe that can be refactored away?